README.md

```diff
@@ -1,136 +1,71 @@
+## 1. Update git to reflect the new release
 
-These steps need to be re-run anytime you change target deployment environment.
+1. on your local machine, in the `BraveSensor-Server` repository:
 
-1. Copy public key from the "ODetect Credentials" vault in 1Password into your local
-`BraveSensor-Server/smartthings_rsa.pub` file
+    1. pull the latest code ready for release: `git checkout devenv && git pull origin devenv`
+    
+    1. run `npm ci`
 
-   - If you are deploying to **dev**, copy the value of `Dev - ODetect SmartThings Automation Public Key`
+    1. decide on an appropriate version number for the new version
 
-   - If you are deploying to **prod**, copy the value of `Prod - ODetect2 SmartThings Automation Public Key`
+    1. update CHANGELOG.md by moving everything in `Unreleased` to a section for the new version
+    
+    1. Run the `./build_prod.sh` script. Note the tag of the new image. See the section on [building and pushing a new container](#build-and-push-a-new-container) for more details about this step.
 
+    1. `cd` into the `~/BraveSensor-Server/sensor-helm-chart` directory
 
-### .env files
+        1. Update Chart.yaml with a new `version` and new `appVersion`, if applicable
 
-The .env file contain important secrets such as Twilio credentials, Postgres credentials as well as the IP address of the Env files for the production and development deployments. 
+        1. Update values.yaml by putting the tag of your new container image in the `tag` field of `image`
 
-These steps need to be re-run anytime the `.env` file changes.
+    1. make a new commit directly on `devenv` which updates the changelog and helm chart
+    
+    1. tag the new commit - for example, if the version number is v1.0.0, use `git tag v1.0.0`
 
-1. Copy from the "ODetect Credentials" vault in 1Password into your local 
-`BraveSensor-Server/.env` file
+    1. push the new version to GitHub: `git push origin master --tags`
 
-   - If you are deploying to **dev**, copy the value of "Dev - env file"
+    1. update the `master` branch: `git checkout master && git merge devenv && git push origin master`
 
-   - If you are deploying to **prod**, copy the value of "Prod - env file"
+## 2. Database changes
 
+If your changes involve changes to the database schema, you'll need to run your database migration script on the appropriate database for your environment. See the [Database Migration section](#database-migration).
 
-### Redis cluster IP
+**If the database changes are breaking, think carefully about how to do this. It may be preferable to briefly shut down the server rather than risk undefined behavior from changing the database schema before changing the code that's running**
 
-The cluster IP of the redis deployment is a required parameter for the node application to open a connection to the database. 
 
-These steps need to be re-run anytime the Redis deployment changes.
+## 3. Deploy changes to the Kubernetes Cluster
```

Comment:
Similar to database changes, we should probably have a section for environment variable changes (with a similar warning) and instructions for how to update the secrets.